### ISSUE TITLE
Not working on AMD Ryzen builds

### ISSUE THREAD

**Issue by keir86** [asineth0/checkn1x#50]
Stuck on "checking if device is ready" or "exploit failed"

**Comment by asineth0** [comment]
Yeah, that happens with my B450-F on certain ports too. Try another port.

Other than that though, unless you want to do some debugging & figure out what's causing it, I can't really do much tbh.

**Comment by keir86** [comment]
> Yeah, that happens with my B450-F on certain ports too. Try another port.
> 
> Other than that though, unless you want to do some debugging & figure out what's causing it, I can't really do much tbh.

I've tried all ports but same issue. How exacly would I go about debugging this?

**Comment by asineth0** [comment]
Look in `dmesg` and things like that. Probably most interested in USB-related things. Also run `checkra1n` from the shell in verbose mode, maybe that'll give something helpful.

**Comment by keir86** [comment]
> Look in `dmesg` and things like that. Probably most interested in USB-related things. Also run `checkra1n` from the shell in verbose mode, maybe that'll give something helpful.

I ended up creating a Hackintosh and checkra1n worked perfectly fine through the same AMD hardware, but other mac tools like ipwndfu still fail so I'm not sure what's going on there.

**Comment by Najmul190** [comment]
> I ended up creating a Hackintosh and checkra1n worked perfectly fine through the same AMD hardware, but other mac tools like ipwndfu still fail so I'm not sure what's going on there.

Do you have a tutorial I can follow to do this, as I can't jb either
